bears/xml/XMLBear.py

```diff
@@ -24,10 +26,11 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
-            return self._process_issues(self.stderr_output, filename)
+        if self.stderr_output:
+            self.use_stderr = True
```

Comment:
not sure if this is needed, why do you keep this here?
The usage of `use_stderr` is determined in the `Lint.lint()` function.